builders/get-started/eth-compare/consensus-finality.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Consensus & Finality
+description: A description of the main differences that Ethereum developers need to understand in terms of consensus and finality on Moonbeam.
+---
+
+## Introduction
+
+While Moonbeam strives to be compatible with Ethereum's Web3 API and EVM, there are some important Moonbeam differences that developers should know and understand in terms of consensus and finality.
+
+In short, consensus is a way for different parties to agree on a shared state. As blocks are created, nodes in the network must decide which block will represent the next valid state. Finality defines when that valid state cannot be altered or reversed.
+
+At the time of writing, Ethereum uses a consensus protocol based on [Proof-of-Work (PoW)](https://ethereum.org/en/developers/docs/consensus-mechanisms/pow/), which provides probabilistic finality. On the contrary, Moonbeam uses a hybrid consensus protocol based on [Nominated Proof-of-Stake (NPoS)](https://wiki.polkadot.network/docs/learn-consensus), which provides deterministic finality.
+
+This guide will outline some of these main differences around consensus and finality, and what to expect when using Moonbeam for the first time.
+
+## Ethereum Consensus and Finality
+
+As stated before, Ethereum is currently using a PoW consensus protocol and the longest chain rule, where finality is probabilistic. 
+
+Probabilistic finality means that the probability that a block (and all its transactions) will not be reverted increases as more blocks are built on top of it. Therefore, the higher the number of blocks you wait, the higher the certainty that a transaction will not be re-organized, and consequently reverted. As suggested by [this blog on finality](https://blog.ethereum.org/2016/05/09/on-settlement-finality/) by Vitalik, _"you can wait 13 confirmations for a one-in-a-million chance of the attacker succeeding."_
+
+## Moonbeam Consensus and Finality
+
+Moonbeam uses a block production mechanism based on the [Polkadot NPoS consensus](https://wiki.polkadot.network/docs/en/learn-consensus). In such a mechanism, there are collators and validators. [Collators](https://wiki.polkadot.network/docs/en/learn-collator) maintain parachains (in this case, Moonbeam) by collecting transactions from users and producing state transition proofs for the Relay Chain [validators](https://wiki.polkadot.network/docs/en/learn-validator). The collators set (nodes that produce blocks) are selected based on the [stake they have in the network](/learn/features/consensus/). 
+
+For finality, Polkadot/Kusama rely on [GRANDPA](https://wiki.polkadot.network/docs/learn-consensus#finality-gadget-grandpa). GRANDPA provides deterministic finality for any given transaction (block). In other terms, when a block/transaction is marked as final, it can't be reverted (only via on-chain governance or forking).
```

Comment:
> it can't be reverted (only via on-chain governance or forking).

Can be rephrased as "it can't be reverted except via on-chain governance or forking." to remove the need for parentheses.